### ISSUE TITLE
Delay should still apply Prefix, Separator

### ISSUE THREAD

**Issue by CurtainUp** [glennreyes/react-countup#335]
While using `useCountUp` with the following parameters, the component renders the plain `start` integer and only adds the prefix and separator once the count begins. 

```js
const AnimatedCounter = () => {
    const { countUp } = useCountUp({
            start: 100,
            end: 1000,
            delay: 1,
            duration: 3,
            prefix: "$",
            separator: ","
        });
    return <div>{countUp}</div>;
};
```

It seems like this should be innate behavior. Am I missing something?

**Comment by mmarkelov** [comment]
@CurtainUp yeah! Seems like a bug, cause right now it doesn't use **prefix**, **suffix** and **separator** props for initialization value:
https://github.com/glennreyes/react-countup/blob/07a97953268c12f58bc635e36c6ae51d57b2f0d4/src/useCountUp.js#L12-L14

I think it will be fixed in #321

**Comment by mmarkelov** [comment]
should be fixed in**react-countup@5.0.0-0**
